### ISSUE TITLE
307406684 Prevalidation N/A (Cheryl)

### ISSUE THREAD

**Issue by FreeREGcomputer** [FreeUKGen/FreeCENMigration#1811]
Issue reported by **Cheggs** at 2024-09-27 22:44:44 UTC
Time: 2024-09-27T22:35:27+00:00
Session ID: 2b9bf58fb48b14a707c7d3546b352098
Problem Page URL: [/manage_counties/select_action?locale=en](/manage_counties/select_action?locale=en)
Previous Page URL: [https://www.freecen.org.uk/manage_counties/new?locale=en](https://www.freecen.org.uk/manage_counties/new?locale=en)
Reported Issue:
I have been doing prevalidation of 1861 POB for LAN for Geoff Jarvis. Yesterday when I went to the list of pieces I saw “prevalidation N/A” and “Man validation N/A” so could not proceed. I checked the lists for a few other counties and their lists were the same. Could someone pls have a look at this. Thanks. Cheryl Meggers.   chemeggs@yahoo.com,au
    

![Screenshot](https://www.freecen.org.uk)

**Comment by AnneV-Learn** [comment]
Old code was inadvertently deployed to Production and this caused an issue with the CSV Upload/processing of a CSV file that had commenced Validation. I reverted the appropriate code and also reverted the code that was disabling VLD Pre-Validaton which had been implemented in an attempt to resolve the issue but I believe there are not actually issues with VLD Pre-processing. I have tested on Test3 - seems OK.
I believe this is ready for Deployment to Production now.
